<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1_Animation_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button 1 Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2_Animation_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button 2 Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3_Animation_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button 3 Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4_Animation_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Button 4 Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>225</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
